libs/safe/src/lib/components/form-modal/form-modal.component.ts

```diff
@@ -232,6 +232,27 @@ export class SafeFormModalComponent
       this.temporaryFilesStorage
     );
 
+    // Set questions readOnly propriety
+    const structure = JSON.parse(this.form?.structure || '');
+    const pages = structure.pages;
+    const initReadOnly = (elements: any): void => {
+      elements?.forEach((question: any) => {
```

Comment:
```suggestion
      elements.forEach((question: any) => {
```
You are already checking if elements exists before calling initReadOnly question, no need for duplicated checks 😜